hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/utils/OzoneManagerRatisUtils.java

```diff
@@ -228,15 +228,34 @@ public static Status exceptionToResponseStatus(IOException exception) {
   }
 
   /**
-   * Obtain Transaction info from downloaded snapshot DB.
+   * Obtain OMTransactionInfo from Checkpoint.
+   */
+  public static OMTransactionInfo getTrxnInfoFromCheckpoint(
+      OzoneConfiguration conf, Path dbPath) throws Exception {
+    Path dbDir = dbPath.getParent();
+    String dbName = dbPath.getFileName().toString();
+    if (dbDir == null) {
+      throw new IOException("Checkpoint {} does not have proper DB location");
```

Comment:
Print the `dbPath` in the exception message to make debugging easier in case this happens.